CONTRIBUTING.md

```diff
@@ -128,10 +128,12 @@ getting the PR accepted as a valid candidate PR for the `master` branch.
 PR authors may also find it useful to run the following script locally in order
 to check that each of the commits within the PR satisfies the requirements
 above, before submitting the PR to review:
+
 ```shell script
-RUSTUP_TOOLCHAIN=1.41.1 ./contrib/test.sh
+RUSTUP_TOOLCHAIN=1.48.0 ./contrib/test.sh
 ```
-Please replace the value in `RUSTUP_TOOLCHAIN=1.41.1` with the current MSRV from
+
+Please replace the value in `RUSTUP_TOOLCHAIN=1.48.0` with the current MSRV from
```

Comment:
This and above don't need changing really, we specifically added the "replace the value ..." bit so we didn't have to maintain these version numbers. It is also totally unrelated to this PR, it doesn't hurt though so I'm happy to let it through.

Reply:
Since I was touching this file, I thought "why not?"

Reply:
If you think it was worth doing it should have been in a separate patch, based on the idea that we try to optimise for review, unrelated changes make it harder to review because the reviewer has to think why they are there. Same goes for the whitespace change in this PR too. Its no biggy here because its the readme, but in a code file this sort of thing matters. This is all quite academic because this project is so small but I still think its worth doing.

Reply:
Let's drop it them...